packages/auth/src/commands/auth/2fa/disable.ts

```diff
@@ -13,6 +13,5 @@ export default class Auth2faGenerate extends Command {
 
   async run() {
     CliUx.ux.error('this command has been removed, in favor of disabling MFA in your Account Settings in a browser: https://dashboard.heroku.com/account')
-    CliUx.ux.exit(1)
```

Comment:
Quick check, you're positive this is unreachable? I'm guessing `CliUx.ux.error` does this as well.

Reply:
Yes to both.